Quadriga/src/main/webapp/resources/js/cytoscape/publicNetwork.js

```diff
@@ -197,6 +213,43 @@ function getTexts(node, path, unixName) {
 	});
 }
 
+function getEntireText(node,path){
+	$(document).ready(function() {
+	$('#txtModal').on('show.bs.modal',function(event){
```

Comment:
either you have a function or you do something on document ready. this does not make sense.